deploy/roles/sas/tasks/main.yml

```diff
@@ -85,3 +97,9 @@
   with_items: "{{ sas_log_file_locations }}"
   tags:
     - sas
+
+- name: Enable sas service
+  service:
+    state: started
+    enabled: yes
+  when: install_sas
```

Comment:
@rioh You don't need the conditional here, since if the service is already started and enabled, this will be a no-op.  It doesn't hurt to ensure these things are true on every run.